scripts/ansible/roles/linux/intel/vars/ubuntu/main.yml

```diff
@@ -8,12 +8,16 @@ intel_sgx_packages:
   - "libsgx-ae-qve"
   - "libsgx-ae-pce"
   - "libsgx-ae-qe3"
+  - "libsgx-qe3-logic"
```

Comment:
Does this also need to be added to the RHEL version of this file just to keep them in sync?

Reply:
I agree with this. However, for RHEL, there isn't YUM repository maintained by Intel (like it's happening for APT). Or, at least, it wasn't when I initially added the RHEL 8 support.

For RHEL / YUM-based platform, what they do instead is: publishing a `tgz` file:
https://github.com/openenclave/openenclave/blob/225391997dfbb663508ef249729e3dcfa38bbca5/scripts/ansible/roles/linux/intel/vars/redhat/ootpa.yml#L6

that you can use to setup a local YUM repo. You download the archive on the system, and add a new YUM repository pointing to the file (the archive contains all the metadata needed to setup a local YUM repo). We take care of this with Ansible.

So, if we need these packages on RHEL, we need to make sure of:
* Check if the `libsgx-qe3-logic` and `libsgx-pce-logic` packages are bundled with the YUM local repo archive that we have (otherwise we need to update YUM local repo `tgz` URL)
* Check if the packages names are the same. Looking at the current packages on both platforms, they might have a slight name difference:
 https://github.com/openenclave/openenclave/blob/225391997dfbb663508ef249729e3dcfa38bbca5/scripts/ansible/roles/linux/intel/vars/redhat/main.yml#L6-L11
 and https://github.com/openenclave/openenclave/blob/225391997dfbb663508ef249729e3dcfa38bbca5/scripts/ansible/roles/linux/intel/vars/ubuntu/main.yml#L6-L11
 Even though most of names are consistent, they might be different sometimes (for example `libsgx-enclave-common-dev` vs `libsgx-enclave-common-devel`)

Reply:
You are both correct in the ideals of keeping in sync, but the rpm packages in question are used for attestation. Currently, without RHEL supporting attestation and only running in simulation mode, they are not needed.

Would it be best to add packages that are not used, or leave them out and add them later?